pkg/dom0-ztools/Dockerfile

```diff
@@ -10,12 +10,17 @@ COPY /patches /
 #  * ZFS on Linux
 ENV ZFS_VERSION=2.1.2
 ENV ZFS_COMMIT=zfs-${ZFS_VERSION}
-ENV ZFS_REPO=https://github.com/openzfs/zfs.git
+# must not end with .git for ADD to work correctly with zipball/
+ENV ZFS_REPO=https://github.com/openzfs/zfs
 ENV ZFS_PATCH_DIR=/patches-zfs-"${ZFS_VERSION}"
 
 WORKDIR /tmp/zfs
 
-RUN git clone --depth 1 -b ${ZFS_COMMIT} ${ZFS_REPO} .
+ADD ${ZFS_REPO}/zipball/${ZFS_COMMIT}/ zfs.zip
```

Comment:
We use tagged version here, probably we can download [tar.gz](https://github.com/openzfs/zfs/archive/refs/tags/zfs-2.1.2.tar.gz) directly?

Reply:
We certainly can. I want to leave it for now, though, to show how you would do a `git clone` when `ADD git...` is not yet available and won't be for a while.

If/when we are ready for this, I'll switch the obvious ones to direct downloads (like you recommend).